tests/v2/validation/rbac/rbac_test.go

```diff
@@ -59,6 +60,19 @@ func (rb *RBTestSuite) SetupSuite() {
 
 }
 
+func checkRBACResourceExistsForUser(client *rancher.Client, collection *v1.SteveCollection, userName string) (bool, error) {
```

Comment:
Do we want the boolean to be returned here as we are not using it anywhere else in the called function?

Reply:
Good catch, we don't need it. Removed it